src/main/java/org/pesmypetcare/webservice/dao/MedicationDao.java

```diff
@@ -0,0 +1,92 @@
+
+package org.pesmypetcare.webservice.dao;
+
+import org.pesmypetcare.webservice.entity.MedicationEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+
+import java.util.List;
+import java.util.Map;
+
+
+
+public interface MedicationDao {
+    /**
+     * Creates a medication eaten by a pet on the database.
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param dateName Date + name of the receipt of the medication
+     * @param medication The medication entity that has the attributes of the medication for the pet.
+     */
+    void createMedication(String owner, String petName, String dateName,MedicationEntity medication);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [',' is not followed by whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5348142)